src/osd/PrimaryLogPG.cc

```diff
@@ -13206,9 +13208,6 @@ uint64_t PrimaryLogPG::recover_backfill(
     }
   }
 
-  hobject_t backfill_pos =
```

Comment:
Same here, explain why you are doing this for future git-blame readers.

Reply:
The shuffling was only to move closer to the usage. Added a note to the commit message saying this is intended to be a refactoring only.